### PR TITLE
BitSum.ParkControl version 1.5.0.10

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28511)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
what

**Comment by BajajAditya123** [comment]
@OfficialEsco After commiting your changes this is happening now what to do?

File: manifests/b/Bitsum/ParkControl/1.5.0.10/Bitsum.ParkControl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Bitsum.ParkControl
```
Comment (OfficialEsco) [inline manifests/b/Bitsum/ParkControl/1.5.0.10/Bitsum.ParkControl.installer.yaml:4]:
```suggestion
PackageIdentifier: BitSum.ParkControl
```

File: manifests/b/Bitsum/ParkControl/1.5.0.10/Bitsum.ParkControl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Bitsum.ParkControl
```
Comment (OfficialEsco) [inline manifests/b/Bitsum/ParkControl/1.5.0.10/Bitsum.ParkControl.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: BitSum.ParkControl
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Capitalization issue, please change the folder name and file names from Bit**s**um to Bit**S**um.

File: manifests/b/Bitsum/ParkControl/1.5.0.10/Bitsum.ParkControl.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Bitsum.ParkControl
```
Comment (OfficialEsco) [inline manifests/b/Bitsum/ParkControl/1.5.0.10/Bitsum.ParkControl.yaml:4]:
```suggestion
PackageIdentifier: BitSum.ParkControl
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
The bot finally detects if a PR is about to break the Repo.
```
The file was not found. Verify git path must match with the casing of the file in disk. Not found file on 'manifests\b\Bitsum\ParkControl\1.5.0.10\Bitsum.ParkControl.installer.yaml'
```

Stop submitting multiple PRs when you can fix the changes in the current PR.

**Comment by BajajAditya123** [comment]
it should be fine now

**Comment by OfficialEsco** [comment]
You will have to change the Publisher folder name.
`manifests/b/Bitsum/ParkControl/1.5.0.10/`
to
`manifests/b/BitSum/ParkControl/1.5.0.10/`

**Comment by BajajAditya123** [comment]
how to do that

**Comment by OfficialEsco** [comment]
For the third time, press `.` on your keyboard.

**Comment by BajajAditya123** [comment]
opened web editor

**Comment by BajajAditya123** [comment]
image

**Comment by OfficialEsco** [comment]
Click the icon in the Top Right corner named Explorer, there you can work with the files just like File Explorer.

**Comment by BajajAditya123** [comment]
folder BitSum already created

**Comment by OfficialEsco** [comment]
> Drag ParkControl to BitSum then Capitalize the S in the installer, version and locale.

**Comment by BajajAditya123** [comment]
image

**Comment by BajajAditya123** [comment]
I'm doing it

**Comment by BajajAditya123** [comment]
I think so

**Comment by OfficialEsco** [comment]
Yes do that, then rename the files, then when you are done with all of that you click the third icon on the left, the click the checkmark that says commit and it will appear here.

**Comment by BajajAditya123** [comment]
image

**Comment by OfficialEsco** [comment]
Type Resolved Capitalization in the Message field

**Comment by BajajAditya123** [comment]
what to do now

**Comment by BajajAditya123** [comment]
image
now its ok

**Comment by OfficialEsco** [comment]
Uhhh, hold on, is there a Push button somewhere?

**Comment by BajajAditya123** [comment]
I don't think so

**Comment by BajajAditya123** [comment]
I checked every icon on the screen

**Comment by OfficialEsco** [comment]
Ohh you have to click Commit after you've typed your commit message

**Comment by BajajAditya123** [comment]
done

**Review by ItzLevvie — APPROVED** [review]

**Comment by BajajAditya123** [comment]
Now I understand how mods work
The wingetbot and msftbot just check that it installs in background and when the validation ssucced you mods than check the code and then review and then it merges to the repo

**Pull request merged**